prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -202,7 +216,13 @@ for det in `echo $LIST_OF_DETECTORS | sed "s/,/ /g"`; do
     has_detector_reco $det && add_comma_separated TRACK_SOURCES "$det"
   fi
 done
-PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+
+if [ -z $ISPILOTBEAMDATA ]; then
+  PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+else
+  PVERTEX_CONFIG="--vertexing-sources ITS --vertex-track-matching-sources ITS --hbfutils-config none"
```

Comment:
I think it would be more flexible if we changed it in this way:
```
[[ -z "VERTEXING_SOURCES" ]] && VERTEXING_SOURCES="$TRACK_SOURCES"
PVERTEX_CONFIG="--vertexing-sources $VERTEXING_SOURCES --vertex-track-matching-sources $VERTEXING_SOURCES"
```
Then one could set externally `VERTEXING_SOURCES=ITS` for pilot beam data.

Reply:
Done